configurations/amp-ad/logoHeaderConfig.ts

```diff
@@ -1,5 +1,5 @@
 import { LogoConfig } from '../types/portal-config'
-import ampAdLogoSvg from './style/amp-ad-logo.svg'
+import ampAdLogoSvg from './style/amp-logo.svg'
```

Comment:
there is actually an svg w/ that name?

Reply:
yep! you can see the change here - https://github.com/Sage-Bionetworks/portals/pull/101/files#diff-d7af22d7a81d671b4f31a68c1dca78cd